pytensor/link/pytorch/dispatch/elemwise.py

```diff
@@ -11,9 +11,21 @@ def pytorch_funcify_Elemwise(op, node, **kwargs):
     scalar_op = op.scalar_op
     base_fn = pytorch_funcify(scalar_op, node=node, **kwargs)
 
-    def elemwise_fn(*inputs):
-        Elemwise._check_runtime_broadcast(node, inputs)
-        return base_fn(*inputs)
+    if hasattr(scalar_op, "nfunc_spec") and hasattr(torch, scalar_op.nfunc_spec[0]):
```

Comment:
Should do the same trick for `scipy.x` you did in another PR?

Reply:
...but then I have to write a new test for this PR 😅